lib/mindwendel_web/live/label_live/captions_component.html.heex

```diff
@@ -3,7 +3,7 @@
     <div class="m-1">
       <span
         class="badge rounded-pill"
-        id="idea-caption-#{uuid()}"
+        id={"idea-caption-#{uuid()}"}
```

Comment:
this one still needs a regression test

Reply:
ok so its still WIP?

Reply:
kinda :sweat_smile:  Depends on how important you think that regression test is, but I wanted to test it yeah and it felt too small to split off even for me. However, it can also be merged and we ship the regression test later.

Reply:
ok then lets merge this